net/ampr-ripd/Makefile

```diff
@@ -48,48 +49,27 @@ define Package/ampr-ripd/postrm
 [ -z "$${IPKG_INSTROOT}" ] || exit 0
 
 echo "Removing firewall rules..."
-for i in $$(seq 99 -1 0); do
-   if [ $$(uci -q get firewall.@rule[$$i]) ]; then
-      name=$$(uci get firewall.@rule[$$i].name)
-      if [ "$$name" = "Net 44 ICMP Echo Request" ] \
-         || [ "$$name" = "Net 44 Router ICMP" ]    \
-         || [ "$$name" = "ipip" ]; then
-         uci del firewall.@rule[$$i]
-      fi
-   fi
-done
+for i in $$(uci show firewall | awk -F= '/@rule/ && /name='\''(Net 44 ICMP Echo Request|Net 44 Router ICMP|ipip)'\''/ {split($$1, conf, /\./); print conf[2]}' | sort -r); do
+      uci del firewall."$$i"
+   done
```

Comment:
Usually the start (`for`) and the end (`done`) of a loop are at the same indentation level. Same goes for other loops.

I assume I wasn't clear enough in my last comment wrt double quotes. What I meant was that `awk` rules are more readable when double-quoted, so you don't have `'\''`. Mind you, `\$$1` will need to be escaped.